src/app/components/left-panel/left-panel.component.html

```diff
@@ -7,48 +7,48 @@
       </div>
     </li>
     <li>
-      <a class="btn btn-dark w-100 text-left" href="">
-        <i class="fas fa-home"></i> Home
+      <a routerLink="/home" class="btn btn-dark w-100 text-left">
+        <i class="fas fa-home"></i> home
       </a>
     </li>
     <li>
-      <a class="btn btn-dark w-100 text-left" href="">
-        <i class="fas fa-hammer"></i> Service catalog
+      <a routerLink="/service-catalog" class="btn btn-dark w-100 text-left">
+        <i class="fas fa-hammer"></i> service catalog
       </a>
     </li>
     <li>
-      <a class="btn btn-dark w-100 text-left" href="">
-        <i class="fas fa-users"></i> Panel users
+      <a routerLink="/panel-users" class= "btn btn-dark w-100 text-left">
```

Comment:
Remove the white space between class and value **class= "btn btn-dark w-100 text-left"**